docs/docker/Dockerfile.bionic

```diff
@@ -72,3 +63,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # 9. Run everything after as non-privileged user.
 USER pwuser
+
+# === BAKE BROWSERS INTO IMAGE ===
+
+# 1. Add tip-of-tree Playwright package to install its browsers.
+#    The package should be built beforehand from tip-of-tree Playwright.
+COPY --chown=pwuser ./playwright.tar.gz /home/pwuser/playwright.tar.gz
```

Comment:
Can we omit this step and just install playwright (@next?) from npm?

Reply:
1. it would be racy
2. it won't work for release builds